examples/website-demos/virtual-event/src/components/CentralPanel/RoomsList.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from 'react';
+
+import { EventCard } from './EventCard';
+import { rooms } from './rooms';
+
+import { CalendarButton } from '../../assets/CalendarButton';
+
+import { useEventContext } from '../../contexts/EventContext';
+
+import './RoomsList.scss';
+
+export const RoomsList = () => {
+  const [dropdown, setDropdown] = useState(false);
+  const { chatType } = useEventContext();
+
+  const calendarClick = () => {
+    setDropdown(!dropdown);
+  };
+
+  return (
+    <>
+      <div className={`rooms-list-video${chatType === 'room' ? '' : '-hidden'}`}>
+        <h1>ROOM VIDEO</h1>
+      </div>
+      <div className={`rooms-list-container${chatType === 'room' ? '-hidden' : ''}`}>
+        <div className='rooms-list-header'>
+          <div className='rooms-list-header-title'>
+            <div className='rooms-list-header-title-main'>World Hacker Summit 2021</div>
+            <div className='rooms-list-header-title-sub'>Stream.IO</div>
```

Comment:
i know this in the designs but we're not stream.io, maybe chance to presented by Stream